src/SFA.DAS.FAT.Web/Views/Shared/_Layout.cshtml

```diff
@@ -119,6 +119,12 @@
                         </a>
                     </li>
 
+                    <li class="govuk-footer__inline-list-item">
+                        <a class="govuk-footer__link" asp-controller="Accessibility" asp-action="AccessibilityStatement" target="_blank" rel="noreferrer noopener">
```

Comment:
```suggestion
                        <a class="govuk-footer__link" asp-route="@RouteNames.AccessibilityStatement" target="_blank" rel="noreferrer noopener">
```

Can we do the above changes and then change the link to the following, using the asp-route attribute?